src/components/Documents/DocumentTable.jsx

```diff
@@ -50,7 +52,7 @@ const DocumentTable = () => {
           </TableHead>
           <TableBody>
             {documentListObject?.docList.map((document) => (
-              <DocumentTableRow key={document.name} document={document} />
+              <DocumentTableRow key={document.name} document={document} user={user} />
```

Comment:
I see we were already using `document.name` as key here.

Do we have some sort of validation that a file can't be uploaded with same name as one that already exists OR it can, but it overwrites the original - so this won't be an issue?

Reply:
Looking into it, I believe it overwrites the original.  If that is the case, then this is fine.